<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalLevelConfForm</class>
 <widget class="QDialog" name="GlobalLevelConfForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPlainTextEdit" name="ProloguePlainTextEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>391</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="EpiloguePlainTextEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>391</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="OkButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>620</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>620</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Level number</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="LevelNumSpinBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>42</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Level Prologue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>450</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Level Epilogue</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
